lonestar/liblonestar/include/Lonestar/Search.h

```diff
@@ -0,0 +1,54 @@
+
+
```

Comment:
Can't these routines be part of utils.h? search.h is slightly confusing.

Reply:
Ok. Will rename it as utils.h

Reply:
There is already a utils.h somewhere is what Gurbinder is saying

Reply:
Not in liblonestar. I don't want to move this to libgalois or anywhere else.